specifyweb/frontend/js_src/css/workbench.css

```
@@ -28,14 +28,30 @@
   @apply leading-5;
 }
 
+.handsontable .colHeader.columnSorting {
+  @apply inline-flex items-center gap-1 overflow-hidden whitespace-nowrap;
+  min-width: 0;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)